cst/plugins/eu.esdihumboldt.cst.functions.numeric/src/eu/esdihumboldt/cst/functions/numeric/sequentialid/SequentialID.java

```diff
@@ -78,16 +79,21 @@ protected Object evaluate(String transformationIdentifier, TransformationEngine
 			break;
 		}
 
-		long id;
+		long id = 1;
 		synchronized (context) {
 			Long seqValue = (Long) context.get(key);
 			if (seqValue != null) {
 				id = seqValue + 1;
 			}
 			else {
-				id = START_VALUE;
+				try {
+					id = Integer.parseInt(startValue);
+					context.put(key, id);
+				} finally {
+					System.out.println("ERROR with the input value of startValue  " + startValue
+							+ " for ID creation: the input value should be an integer");
+				}
 			}
-			context.put(key, id);
```

Comment:
This line needs to still apply both for if and else case as before - otherwise you won't have a sequence but always the start value used.